modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/bookmark/BookmarkActivity.kt

```diff
@@ -0,0 +1,31 @@
+package au.com.shiftyjelly.pocketcasts.player.view.bookmark
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.WindowInsetsControllerCompat
+import dagger.hilt.android.AndroidEntryPoint
+import au.com.shiftyjelly.pocketcasts.views.R as VR
+
+@AndroidEntryPoint
+class BookmarkActivity : AppCompatActivity() {
```

Comment:
Can we put the comment you added to the PR description explaining why we need this activity?